### PR TITLE
fix: RSS-ECOMM-2_17-Add responsive styles for navigation. Add favicon.

### PR DESCRIPTION
**Pull request by Pavel-Kuvshinov**
## Overview
Please provide a brief summary of what this pull request accomplishes. Include the motivation behind these changes and the impact they aim to have on the project.

**What has been done:**
- Add responsive styles for header and navigation.
- Add favicon.

**Why this has been done:**
- The changes improve the display of the project on various devices

**Issue(s) closed:**
- [Issue RSS-ECOMM-2_17: Add links to all functional pages on the main page](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/eCommerce-Application/Sprints/Sprint2/RSS-ECOMM-2_17.md)

## Type of Change
- [х] New feature (non-breaking change which adds functionality)
- [х] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Problem Areas
- It is necessary to check the favicon display when publishing the project

## Checklist:
- [х] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: kickFlip-project/src/components/app/app.tsx [hunk 1]
```diff
@@ -1,26 +1,66 @@
...
 import { useEffect } from 'react';
-import RegistrationPage from '@/pages/registration/registrationPage';
 import { useDispatch } from '@/services/store';
 import { getAnonymousToken, getUser } from '@/services/userSlice';
 import { getCookie } from '@/utils/cookie';
 
+import HomePage from '@/pages/home/homePage';
+import LoginPage from '@/pages/login/loginPage';
+import RegistrationPage from '@/pages/registration/registrationPage';
+import NotFoundPage from '@/pages/notfoundpage/notfoundPage';
```
Comment (makarovaiuliia) [inline kickFlip-project/src/components/app/app.tsx]: notFoundPage

File: kickFlip-project/src/pages/notfoundpage/notfoundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom';
+import './notfoundPage.css';
```
Comment (makarovaiuliia) [inline kickFlip-project/src/pages/notfoundpage/notfoundPage.tsx]: notFoundPage

**Review by makarovaiuliia — APPROVED** [review]
Вот тут надо исправить пути

File: kickFlip-project/src/components/app/app.tsx (hunk 1 above)
Comment (Pavel-Kuvshinov) [reply kickFlip-project/src/components/app/app.tsx]: Испарвил

File: kickFlip-project/src/pages/notfoundpage/notfoundPage.tsx (hunk 2 above)
Comment (Pavel-Kuvshinov) [reply kickFlip-project/src/pages/notfoundpage/notfoundPage.tsx]: Исправил

**Review by makarovaiuliia — APPROVED** [review]

**Pull request merged**